<launch>
    <arg name = "drone_id" default = "1"/>
    <arg name = "takeoff_height" default = "0.6"/>
    <arg name = "in_real" default="true" />

    <include if="$(arg in_real)" file="$(find mavros)/launch/px4.launch">
        <arg name="fcu_url" value="/dev/ttyACM0:921600"/>
        <!-- <arg name="fcu_url" value="udp://:14550@127.0.0.1:18570"/> -->

        <arg name="gcs_url" value="udp://:14556@10.42.0.200:14550"/>
    </include>

    <node name = "ctrl_bridge" pkg = "control_for_gym" type = "ctrl_bridge" output = "screen">
        <remap from="~vel_cmd" to="/airgym/cmd"/>
        <param name ="drone_id" value="$(arg drone_id)"/>
        <param name ="takeoff_height" value="$(arg takeoff_height)"/>

    </node>
</launch>
